### PR TITLE
Fix filter by map/atlas if multiple plots in layouts

### LINKED ISSUE #244: Overlay two graphics on atlas
**Describe the bug**

I am trying to overlay two graphics in the atlas. 
I think that should be done from element properties by adding two graphics (screenshot 1).

I have an atlas with 100 points. I am interested in representing in the same graphic the 100 points and in another color (superimposing) the point to which the atlas refers. As you can see in the screenshot 2 I have represented 2 times the two points but in the screenshot 3 you can see all the points. 
I have activated "use only features inside atlas feature" in onli one but it seems that activating this box affects both graphics and not each one independently.
It is a bug or I am wrong about something?
Thanks 

C1
----------------------------------------------------------------------
C2
----------------------------------------------------------------------------------------------------------
C3

### PR DESCRIPTION
**Pull request by SGroe**
Fixes #244 

I moved the PlotLayoutItem attributes filter_by_map and filter_by_atlas to the PlotSettings to use these settings per plot.

### CODE REVIEW

File: DataPlotly/gui/layout_item_gui.py [hunk 1]
```diff
@@ -244,10 +254,13 @@ def setNewItem(self, item):  # pylint: disable=missing-docstring
 
             self.panel.filter_by_map_check.blockSignals(True)
             self.panel.filter_by_map_check.setChecked(item.filter_by_map)
```
Comment (jdlom) [inline DataPlotly/gui/layout_item_gui.py:256]: I think this line should be removed

**Comment by jdlom** [comment]
@SGroe I just have tested and it works. I think there is an irrelevant error with the line 256  you need to delete.

File: DataPlotly/gui/layout_item_gui.py (hunk 1 above)
Comment (SGroe) [reply DataPlotly/gui/layout_item_gui.py]: Done!

**Comment by SGroe** [comment]
@nyalldawson do you know why all checks fail? There are timeouts after 10 minutes ...

**Comment by nyalldawson** [comment]
@SGroe again, ignore the travis result -- but the other Github action test failures are legitimate issues with the PR

**Comment by SGroe** [comment]
This is ready to merge. Another review after the latest commits would be great.

**Review by ghtmtt — APPROVED** [review]
LGTM! I tested the PR and it works like a charm

**Pull request merged**